crates/telio-proxy/src/proxy.rs

```diff
@@ -523,6 +531,9 @@ impl Runtime for StateEgress {
             (pk, _, _)= update => {
                 self.replace_socket(pk).await;
             },
+            else => {
+                telio_log_warn!("StateEgress: no events to wait on ...");
```

Comment:
I think you could improve this message a bit, you don't know whether there are any events when you are waiting.

Reply:
What it should say?